### PR TITLE
AC Automated Test - LRAC-11825 | master

### PR DESCRIPTION
**Pull request by ViniLopes20**
https://issues.liferay.com/browse/LRAC-11825 (Automation Ticket)
https://issues.liferay.com/browse/LRAC-11573 (Bug)
[Testray Result](https://testray.liferay.com/home/-/testray/case_results?testrayBuildId=1960680677&testrayRunId=1960728623)

I needed to create a new function for the `MouseOver.function` file because none of the other already existing functions were working for the chart I was using

### CODE REVIEW

**Comment by ViniLopes20** [comment]
ci:test:sf

File: portal-web/test/functional/com/liferay/portalweb/functions/MouseOver.function [hunk 1]
```diff
@@ -50,6 +50,20 @@ definition {
 		selenium.assertLiferayErrors();
 	}
 
+	function mouseUp {
```
Comment (joshitagaki) [inline portal-web/test/functional/com/liferay/portalweb/functions/MouseOver.function:53]: That's odd that no mouseover function works for this bar graph. Would this do the trick instead: https://github.com/liferay/liferay-portal/blob/master/portal-web/test/functional/com/liferay/portalweb/functions/MouseUp.function#L4

Looks similar to what you want to implement

Comment (ViniLopes20) [reply portal-web/test/functional/com/liferay/portalweb/functions/MouseOver.function:53]: Good catch, Josh. I will check if this works for us

Comment (ViniLopes20) [reply portal-web/test/functional/com/liferay/portalweb/functions/MouseOver.function]: Hey Josh, your suggestion is working I updated the PR. Thank you for that

**Review by joshitagaki — APPROVED** [review]
Very nice! LGTM!

**Review by manuferreira — APPROVED** [review]
LGTM! Thank you, Vini. Nice that you found a function :tada:

**Review by christianstokes — APPROVED** [review]
LGTM. Thanks Vini, great test.

Approved.

**Comment by christianstokes** [comment]
ci:forward

**Pull request closed without merging**